### ISSUE TITLE
Wrong link in doc

### ISSUE THREAD

**Issue by ottopaulsen** [intlify/vue-i18n-next#1220]
### Reporting a bug?

The link to Composition API on [this page](https://vue-i18n.intlify.dev/guide/#javascript) in the doc points to the Vue doc.

### Expected behavior

It should point to the [Composition API](https://vue-i18n.intlify.dev/guide/advanced/composition.html) page for vue-i18n.

### Reproduction

Go to [this page](https://vue-i18n.intlify.dev/guide/), scroll to the bottom, click the link to `Composition API`.

### System Info

```shell
N/A
```

### Screenshot

<img width="689" alt="image" src="https://user-images.githubusercontent.com/3034647/201334569-125b9efa-f1ad-4f10-87e8-cfb3b5e783fa.png">

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/vue-i18n-next/blob/master/.github/CONTRIBUTING.md)
- [X] Read the [Documentation](https://vue-i18n.intlify.dev/)
- [X] Check that there isn't [already an issue](https://github.com/intlify/vue-i18n-next/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/intlify/vue-i18n-next/discussions)
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by kazupon** [comment]
Thank you for your feedback!

I think no issue for linking to the composition API in the documentation. I don't think it's a problem to link to the composition API in the documentation, because people don't know about it.
I think the issue is vue-i18n documentation structure.
It would be nice to have a link to Vue I18n's CompositionAPI on that page, and a link to Vue's Composition API in the tip.

**Comment by ottopaulsen** [comment]
I disagree. Look at the text:

> To use similar ways at the setup function, you need to call the useI18n functions. We will learn more about this in the [Composition API](https://v3.vuejs.org/guide/composition-api-introduction.html)

The text is referring to usiI18n functions, saying "We will learn more about this" in the Composition API. You do NOT learn more about useI18n functions in the Vue documentation. You do in the vue-i18n documentation. That is why the link should be to the vue-i18n documentation. The current link is clearly wrong. I discovered this as a user trying to learned these things, and when I clicked the link I did not get where I expected nor needed.
